packages/yourloops/components/date-pickers/calendar-header.tsx

```diff
@@ -78,49 +91,56 @@ const calendarStyles = makeStyles(() => {
     monthTranslated: {
       transform: "translateX(-100%)",
     },
+    buttonHidden: {
+      // When using range calendar, button may be hidden
+      // depending on the calendar position
+      visibility: "hidden",
+    },
   };
 }, { name: "date-pickers-calendar-header" });
 
 function CalendarHeader(props: CalendarHeaderProps): JSX.Element {
-  const { currentMonth, onMonthChange, changeMonth, minDate, maxDate } = props;
+  const { currentMonth, changingMonth, position, onPrevMonth, onNextMonth } = props;
 
   const { t } = useTranslation("yourloops");
   const classes = calendarStyles();
   const animClasses = animationStyle();
 
   const displayFormat = t("date-picker-header-date-format");
   const currMonthYear = currentMonth.format(displayFormat);
-  const prevMonth = currentMonth.startOf("month").subtract(1, "month");
-  const nextMonth = prevMonth.add(2, "month");
-  const prevButtonDisabled = isDayjs(minDate) && prevMonth.isBefore(minDate.startOf("month"));
-  const nextButtonDisabled = isDayjs(maxDate) && nextMonth.isAfter(maxDate);
+  const prevButtonDisabled = typeof onPrevMonth !== "function";
+  const nextButtonDisabled = typeof onNextMonth !== "function";
```

Comment:
True, but it's more explicit this way, it help to understand the code.
Also this will only remove one line of code, I prefer to have a more understandable code in this case.